packages/ariakit/src/focus-trap/__examples__/focus-trap/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FocusEvent, useRef } from "react";
+import { Checkbox, useCheckboxState } from "ariakit/checkbox";
+import { FocusTrap } from "ariakit/focus-trap";
+import "./style.css";
+
+export default function Example() {
+  const focusTrapped = useCheckboxState({ defaultValue: true });
+  const firstRef = useRef<HTMLButtonElement>(null);
+  const lastRef = useRef<HTMLButtonElement>(null);
+
+  const onTrapFocus = (event: FocusEvent) => {
+    if (event.relatedTarget === firstRef.current) {
+      lastRef.current?.focus();
+    } else {
+      firstRef.current?.focus();
+    }
+  };
+
+  return (
+    <>
+      {focusTrapped.value && <FocusTrap onFocus={onTrapFocus} />}
+      <div className="wrapper">
+        <label className="label">
+          <Checkbox state={focusTrapped} clickOnEnter className="checkbox" />
+          Trap focus
+        </label>
+        <button className="button" ref={firstRef}>
+          First
+        </button>
+        <button className="button" ref={lastRef}>
+          Last
+        </button>
+      </div>
```

Comment:
The behavior is a little bit weird now when the focus is trapped. When pressing Tab, the focus never goes to the checkbox, when pressing Shift+Tab, the focus never goes to the last button.

I think we can just make the checkbox the first item ref to fix this:
```suggestion
      <div className="wrapper">
        <label className="label">
          <Checkbox state={focusTrapped} ref={firstRef} className="checkbox" />
          Trap focus
        </label>
        <button className="button" ref={lastRef}>
          Button
        </button>
      </div>
```

Reply:
Thank you so much @diegohaz !! Sorry for the delay! I was able to update the example with your suggestion. I've also added the focus trap test. The Ariakit test utils are really great! ❤️